plugins/org.locationtech.udig.libs/.classpath

```diff
@@ -114,7 +110,6 @@
 	<classpathentry exported="true" kind="lib" path="lib/gt-xsd-wmts-22.1.jar"/>
 	<classpathentry exported="true" kind="lib" path="lib/gt-xsd-wps-22.1.jar"/>
 	<classpathentry exported="true" kind="lib" path="lib/h2-1.1.119.jar"/>
-	<classpathentry exported="true" kind="lib" path="lib/hatbox-1.0.b11.jar"/>

```

Comment:
wondering why hatbox has been removed from classpath, which updated added recently with #458 

Please check, if the commit is based on the latest commit from master